modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,264 @@ Runtime Fabric enables you to specify the level of severity of the message writt
-. Click on the Pods tab.
-. Select the environment ID where the application was deployed on the right dropdown, near the search input.
-. Find the Pod name which begins with the name of your application.
-. Click on the Pod name and select "Logs".
+* Anypoint Monitoring
+* Third party logging solutions using Fluent Bit log forwarding output plugins:
 
-The page should redirect to the Logs tab with a filter applied to your application.
+** Azure
+** Elasticsearch
+** Graylog Extended Log Format (GELF)
+** Splunk
 
 [NOTE]
-To view the latest logs, click the "Refresh" button on the upper right portion of the page.
+Log data can be delivered to multiple destinations. However, only one Fluent Bit log forwarding output plugin is supported at any one time. Multiple log forwarding output configurations are not supported. For example, you can forward logs to Anypoint Monitoring and a third party logging solution, but not to two third party logging solutions.
 
-=== Filters
+=== Enable Log Forwarding
 
-There are two levels of filters to help drill down on the logs to make visible:
+. Install and configure the applicable third party logging solution. Verify that you have allocated the required resources for the solution. Refer to the documentation for your third party logging solution for details. 
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Runtime Fabrics* in the sidebar navigation.
+. Select the applicable Runtime Fabric based on the name used during installation.
+. Select the *Log Forwarding* tab.
+. Select the *Forward to Anypoint Monitoring (Requires Titanium)* checkbox if you have a Titanium subscription and you want to forward logs to Anypoint Monitoring.
+. Select the *Forward to other logging provider* checkbox to forward logs using a Fluent Bit log forwarding output plugin.
+.. In the *Connects to* drop-down list, select the applicable third party log forwarding solution.
+.. Enter Fluent Bit configuration information for your third party logging solution:
 
-* _Containers_ filter on names of containers.
-* _Pods_ filter on the names of pods. This is useful for specifying application names followed by a wildcard (`%`).
+*** <<Azure Log Analytics Configuration Parameters, Azure Log Analytics Configuration Parameters>>
+*** <<Elasticsearch Configuration Parameters, Elasticsearch Configuration Parameters>>
+*** <<GELF Configuration Parameters, GELF Configuration Parameters>>
+*** <<Splunk Configuration Parameters, Splunk Configuration Parameters>>
 
-== Forward Logs to External Services
+.. Select *Submit*.
+.. To verify successful log forwarding, manually check the logs using the external log forwarding service.
 
-Anypoint Runtime Fabric enables you to forward application and cluster logs to an external logging service. The log forwarder built in to Runtime Fabric enables you to send log data to an `rsyslog` server over TCP or UDP.
+==== Azure Log Analytics Configuration Parameters
```

Comment:
This is nesting topic information too deeply, which indicates the need for reorganizing the document topic flow.
Could we have a separate reference topic that includes these, titled something like "Fluent Bit Plugin Configuration Parameters"?

Reply:
BREAK here